bulbs/contributions/filters.py

```
@@ -42,6 +37,21 @@ def apply_published_filter(self, queryset, operation, value):
             raise ValueError("""Publish filters only use before or after for range filters.""")
         return queryset.filter(Published(**{operation: value}))
 
+    def get_start(self, request):
+        start_value = self.get_date_datetime_param(request, "start")
+        if start_value:
+            return timezone.make_aware(start_value).astimezone(timezone.pytz.utc)
+
+    def get_end(self, request):
+        end_value = self.get_date_datetime_param(request, "end")
+        if end_value:
+            end_value += timezone.timedelta(days=1)
+            end_value -= timezone.timedelta(seconds=1)
+            try:
+                return timezone.make_aware(end_value).astimezone(timezone.pytz.utc)
+            except:
+                import pdb; pdb.set_trace()
```

do you want to throw an error or something here instead?